JSBSimForUnreal.vcxproj

```diff
@@ -56,10 +56,10 @@
     <ClCompile>
       <Optimization>Disabled</Optimization>
       <AdditionalIncludeDirectories>$(ProjectDir)src;$(ProjectDir)src\simgear\xml;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>JSBSIM_EXPORT;JSBSIM_VERSION="1.2.0.dev1";WIN32;NOMINMAX;XML_STATIC;_DEBUG;_CONSOLE;HAVE_EXPAT_CONFIG_H;_USE_MATH_DEFINES;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>JSBSIM_EXPORT;JSBSIM_VERSION="1.2.0.dev1";WIN32;NOMINMAX;XML_STATIC;_NDEBUG;_CONSOLE;HAVE_EXPAT_CONFIG_H;_USE_MATH_DEFINES;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Shouldn't the flag `NDEBUG` be added to this list as well ?

According to [C++ docs](https://en.cppreference.com/w/cpp/error/assert), the flag `NDEBUG` need to be defined for the `assert` macro to check and report issues. Unless UE5 defines `NDEBUG` as soon as `_NDEBUG` is defined ?

Reply:
:sweat_smile: Thanks! `_NDEBUG `was typo. Corrected to `NDEBUG`

Reply:
Also if this PR is okay and merged, than Issue #808 is able to be closed